build/tidyconfig.txt

```diff
@@ -1,5 +1,6 @@
 char-encoding: utf8
+drop-proprietary-attributes: no
```

Comment:
It's not in the sense that "no" is the default. Since tidy warns about the use of proprietary attributes that ReSpec uses for definitions and linking (for, lt, dfn-for, link-for), I wanted to make the configuration explicit that these warnings are to be ignored, and protect ourselves in case someone decides to change tidy's default setting for that in a future version.

(I would have hoped that this setting would have made tidy exit with a 0 code as well, but that's not the case)